### PR TITLE
Use newest version of pdpbox (fixes bug on contour_label_font)

### PR DESCRIPTION
**Pull request by dansbecker**
This package hasn't had a release in a while. It has a bug that affects a learn course and which has been fixed on github. Pulling newest version to get that fix.

### CODE REVIEW

**Comment by Philmod** [comment]
This package doesn't seem to have a test file https://github.com/Kaggle/docker-python/tree/master/tests

If a course depends on it, it may be good to add one as part of this PR?

**Comment by rosbo** [comment]
Thanks @dansbecker for adding the test. Is it ready to be merged?

**Comment by dansbecker** [comment]
@Philmod Just added a test. I'm not set up to make it exceptionally easy to test this on our docker image. Is there something I can do to get it tested and verify it passes?

**Comment by rosbo** [comment]
I kicked off a run on the CI server: https://ci.kaggle.net/job/kernels/job/docker-python/job/update-pdpbox/1/

File: tests/test_pdpbox.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from sklearn.model_selection import train_test_split
+from sklearn.tree import DecisionTreeClassifier
+
+class TestPdpbox(unittest.TestCase):
+    def test_simple_pdp(self):
+        # set up data
+        data = pd.read_csv("/input/tests/data/fifa_2018_stats.csv")
+        y = (data['Man of the Match'] == "Yes")
+        feature_names = [i for i in data.columns if data[i].dtype in [np.int64]]
+        X = data[feature_names]
```
Comment (rosbo) [inline tests/test_pdpbox.py]: you need to add an import statement at the top:
```
import numpy as np
```

Comment (rosbo) [reply tests/test_pdpbox.py]: On a linux machine (cloudtop is good), you can run the following to run the tests against the image the CI server built for this PR:
```
cd /path/to/git/docker-python
./test -i gcr.io/kaggle-images/python:update-pdpbox-pretest -p test_pdpbox.py
```

Comment (dansbecker) [reply tests/test_pdpbox.py]: Thanks. The tests pass now

**Comment by rosbo** [comment]
I will merge it as soon as https://ci.kaggle.net/job/kernels/job/docker-python/job/update-pdpbox/2/console completes.

**Comment by rosbo** [comment]
All tests are green. Merging!

**Pull request merged**